index.bs

```diff
@@ -446,30 +459,20 @@ Report Long Animation Frames {#loaf-processing-model}
 
         Note: This can occur if the browser becomes hidden during the sequence.
 
+    1. Let |frameDuration| be the [=duration=] between the [=relative high resolution time=] given |timingInfo|'s [=frame timing info/start time=] and |global|,
+        and the [=current high resolution time=] given |global|.
+
+    1. If |frameDuration| is less than 50 milliseconds, then set |document|'s [=nearest same-origin root=]'s [=current frame timing info=] to null and return.
+
     1. Set |timingInfo|'s [=frame timing info/update the rendering start time=] to |timingInfo|'s [=frame timing info/current task start time=].
 
     1. Set |timingInfo|'s [=frame timing info/style and layout start time=] to |unsafeStyleAndLayoutStart|.
-
-    1. [=report frame timing=] given |document| and the [=unsafe shared current time=].
 </div>
 
 <div algorithm="Queue frame timing">
-    To <dfn export>report frame timing</dfn> given a {{Document}} |document| and a {{DOMHighResTimeStamp}} |unsafeEndTime|:
-
-    1. Let |timingInfo| be |document|'s [=relevant frame timing info=].
-
-    1. Assert: |timingInfo| is not null.
-
-    1. Let |global| be |document|'s [=relevant global object=].
-
-    1. Let |frameDuration| be the [=duration=] between the [=relative high resolution time=] given |timingInfo|'s [=frame timing info/start time=] and |global|,
-        and the [=relative high resolution time=] given |unsafeEndTime| and |global|.
-
-    1. If |frameDuration| is greater than 50 milliseconds, then
-        [=queue a PerformanceEntry|Queue=] a new {{PerformanceLongAnimationFrameTiming}} in |document|'s [=relevant realm=],
-        whose [=PerformanceLongAnimationFrameTiming/timing info=] is |timingInfo|.
-
-    1. set |document|'s [=nearest same-origin root=]'s [=current frame timing info=] to null.
```

Comment:
A lot of steps were removed here-- is this just a simplification, or are these steps mostly duplicated in other places?

Reply:
They are moved to paint timing in the other PR